sorc/ncep_post.fd/CALVOR.f

```diff
@@ -340,9 +350,12 @@ SUBROUTINE CALVOR(UWND,VWND,ABSV)
      &         UWND(I,J+1).LT.SPVAL.AND.UWND(I,J-1).LT.SPVAL) THEN
               R2DX   = 1./(2.*DX(I,J))
               R2DY   = 1./(2.*DY(I,J))
```

Comment:
@JesseMeng-NOAA Do you think these two lines can be commented out?

Reply:
Done